samples/archivist_story_runner.py

```diff
@@ -0,0 +1,305 @@
+"""archivist story runner takes a yaml config file listing operations to be conducted serially.
+
+It then runs the defined operations.
+"""
+
+import time
+from enum import Enum
+import argparse
+import yaml
+
+# remove warning from console
+import urllib3
+
+from archivist.compliance_polices import PolicyType
+from archivist.archivist import Archivist
+
+# stop printing to console that the request is insecure
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```

Comment:
it is up to the user to disable this if he wants to - usually by settuing an environbment variable

PYTHONWARNINGS="ignore:Unverified HTTPS request"

Reply:
I agree!

Have removed